js/main.js

```diff
@@ -4,20 +4,41 @@ var $photo = document.querySelector('.photo');
 var $textInputs = document.querySelectorAll('.text-adjust');
 var $noEntriesText = document.querySelector('.no-entries-text');
 var $formElement = document.querySelector('form');
-var journalFeedList = document.querySelector('ul');
+var $journalFeedList = document.querySelector('ul');
 var $navEntries = document.querySelector('.nav-entries');
 var $navNewEntry = document.querySelector('.new-button');
 var $newEntryForm = document.querySelector('.entry-form');
 var $entriesPage = document.querySelector('.entries');
+var $editTitle = document.querySelector('.edit-form-title');
+var $entryFormTitle = document.querySelector('.entry-form-title');
+var $titleInput = document.querySelector('#title');
+var $commentInput = document.querySelector('#comment');
 
 // Adding event listeners
 $photoUrlInput.addEventListener('input', changeUrl);
 $formElement.addEventListener('submit', formSubmitted);
 window.addEventListener('DOMContentLoaded', addingChildLoop);
 $navEntries.addEventListener('click', navToNewEntry);
 $navNewEntry.addEventListener('click', navToEntries);
+$journalFeedList.addEventListener('click', editEntries);
 
 // function definitions
+function editEntries(event) {
+  if (event.target.className === 'edit-button') {
+    $newEntryForm.setAttribute('class', 'entry-form');
+    $entriesPage.setAttribute('class', 'entries hidden');
+    // Get the entry number from my list item class
+    var entryNumber = event.target.parentNode.parentNode.parentNode.parentNode.className.match(/\d+/g);
+    data.editing = entryNumber[0];
+    $entryFormTitle.className = 'entry-form-title hidden';
+    $editTitle.className = 'edit-form-title';
+    $photoUrlInput.value = data.entries[data.editing - 1].photoUrl;
```

Comment:
You should _never_ use ids for math. As soon as you add the ability to delete entries, this will break colossally.